README.md

```diff
@@ -12,11 +12,43 @@ Kickstart targets *only* the latest stable neovim release (0.7) and the nightly.
 This repo is meant to be used as a starting point for a user's own configuration; remove the things you don't use and add what you miss. This configuration serves as the reference configuration for the [lspconfig wiki](https://github.com/neovim/nvim-lspconfig/wiki).
 
 ### Installation
+
 * Backup your previous configuration
 * Copy and paste the kickstart.nvim `init.lua` into `$HOME/.config/nvim/init.lua`
 * start neovim (`nvim`) and run `:PackerInstall`, ignore any error message about missing plugins, `:PackerInstall` will fix that shortly.
 * restart neovim
 
+### Configuration
+
+You could directly modify the `init.lua` file with your personal customizations. This option is the most straightforward, but if you update your config from this repo, you may need to reapply your changes.
+
+An alternative approach is to create a separate `custom.plugins` module to register your own plugins. In addition, you can handle further customizations in a `after/plugin/defaults.lua` file. See the following examples for more information. Leveraging these files should make upgrading to a newer version of this repo easier. 
```

Comment:
can you add a link to the help for this (since it can be any name, not just `after/plugin/defaults.lua`

Reply:
Okay, I tried to tweak the language to accommodate that. Good point! I'm not sure where I should link, so I added `:help load-plugins`. Is that what you meant?